requirements/product-environment/product-environment.tex

```diff
@@ -0,0 +1,41 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}  
+      \begin{itemize}
+      \item A connection to a LAN with a central server
+      \item 2 MB of storage space and additional 100MB per image
+      \end{itemize}
+    \subsubsection{Software Requirements}  
+      \begin{itemize}
+      \item None, so far
+    \end{itemize}
+
+  \subsection{Central Server}
+    \subsubsection{Hardware Requirements}  
+    \begin{itemize}
+      \item A connection to a LAN
+      \item 500 MB of storage space
+      \item 8 GB of RAM
+      \item One physical CPU core
+    \end{itemize}
+    \subsubsection{Software Requirements}  
+    \begin{itemize}
+      \item Linux Ubuntu 18.04 or newer
```

Comment:
Why particularly Ubunto, it can be any Linux distro.

Reply:
I think we should definitely limit which distributions we officially support because those are the distributions we need to test our software on.
However, Florian specified that our software need to run on CentOS so we should add that to our list of supported distributions.